solr/core/src/java/org/apache/solr/cloud/api/collections/Assign.java

```diff
@@ -325,20 +326,18 @@ public static List<ReplicaPosition> getNodesForNewReplicas(
       ClusterState clusterState,
       String collectionName,
       String shard,
-      int nrtReplicas,
-      int tlogReplicas,
-      int pullReplicas,
+      ReplicaCount numReplicas,
       Object createNodeSet,
       SolrCloudManager cloudManager,
       CoreContainer coreContainer)
       throws IOException, InterruptedException, AssignmentException {
-    log.debug(
-        "getNodesForNewReplicas() shard: {} , nrtReplicas : {} , tlogReplicas: {} , pullReplicas: {} , createNodeSet {}",
-        shard,
-        nrtReplicas,
-        tlogReplicas,
-        pullReplicas,
-        createNodeSet);
+    if (log.isDebugEnabled()) {
+      log.debug(
+          "getNodesForNewReplicas() shard={}, {} , createNodeSet={}",
+          shard,
+          numReplicas.toDebugString(),
```

Comment:
Might it be reasonable for ReplicaCount.toString() to be this debug string?

Reply:
It sounds reasonable, done.